### ISSUE TITLE
White typography disappears in Text interface

### ISSUE THREAD

**Issue by haswalt** [airbnb/Lona#15]
Because the background is white when rendering white text you cannot see the text any more.

I'm guessing either us a contrasting background for white text?

**Comment by dabbott** [comment]
Yeah, that sounds reasonable. Looks like that's what Sketch does:

![screen shot 2017-11-25 at 9 45 00 am](https://user-images.githubusercontent.com/1198882/33233069-64c08cc8-d1c5-11e7-84e1-3a1351c7a339.png)

I think first we'd want to make the text style picker look more like the color picker though, since the text style picker is pretty poorly designed in general right now.

**Comment by haswalt** [comment]
@dabbott I've been thinking about this and have one issue with maintaining the style including the font size and actually this still looks and feels pretty clunky in the sketch UI if you have large font sizes. Although they do cap the font size to a maximum when showing the font styles.

**Comment by dabbott** [comment]
@haswalt we can probably figure out what to do about this now that https://github.com/airbnb/Lona/issues/20 has closed
